### PR TITLE
LPS-179186 Modify ObjectFieldSettings on the ObjectFieldBase file using a filter to stateFlow

### PR DESCRIPTION
**Pull request by vngoliveira**
Related issue [LPS-179186](https://issues.liferay.com/browse/LPS-179186)

### CODE REVIEW

**Comment by vngoliveira** [comment]
ci:test

**Comment by vngoliveira** [comment]
ci:test:sf

**Comment by vngoliveira** [comment]
ci:test:stable

**Comment by vngoliveira** [comment]
ci:test:relevant

**Comment by vngoliveira** [comment]
ci:test:object

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/EditObjectField.tsx [hunk 1]
```diff
@@ -64,6 +64,7 @@ export default function EditObjectField({
 	const onSubmit = async ({id, ...objectField}: ObjectField) => {
 		delete objectField.listTypeDefinitionId;
 		delete objectField.system;
+		delete objectField.objectFieldSettings;
```
[the remark this answers is not in the document]
Comment (vngoliveira) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/EditObjectField.tsx:67]: Ok! I already knew that my solution was a bit radical and the problem was with the `stateflow`, but I couldn't think of a better solution. Thank you for the light!

**Comment by vngoliveira** [comment]
ci:test:sf

**Comment by vngoliveira** [comment]
ci:test:stable

**Comment by vngoliveira** [comment]
ci:test:relevant

**Review by danvb112 — APPROVED** [review]
LGTM!

This PR is blocked because must wait https://github.com/brianchandotcom/liferay-portal/pull/132662. It is possible that this PR that is in Brian's queue has fixed this bug, if that is the case we will close this PR

**Comment by samuelromeiro** [comment]
Brian merged the https://github.com/brianchandotcom/liferay-portal/pull/132662. 
I'm closing this PR and I'm going to ask a QA to validate if the https://github.com/brianchandotcom/liferay-portal/pull/132662 fixes this bug.

**Pull request closed without merging**